wifi.go

```diff
@@ -304,3 +304,38 @@ func parseIEs(b []byte) ([]ie, error) {
 
 	return ies, nil
 }
+
+type SurveyInfo struct {
+	// The frequency in MHz of the channel.
+	Frequency int
+
+	// The noise level in dBm.
+	Noise int
+
+	// The time in milliseconds the radio has spent on this channel.
```

Comment:
This is a `time.Duration`, which is actually nanoseconds. Since this is being normalized to a `time.Duration`, it's not actually milliseconds.

```suggestion
	// The time the radio has spent on this channel.
```

Reply:
Removed milliseconds from all the comments